gtfs/spec/en/reference.md

```diff
@@ -94,41 +95,41 @@ The following example demonstrates how a field value would appear in a comma-del
 
 File: **Required**
 
-|  Field Name | Required | Details |
+|  Field Name | Required & Type | Details |
```

Comment:
I would separate "Type" into it's own column.  GTFS-realtime has:

Field Name | Type | Required | Cardinality | Description
-- | -- | -- | -- | --

For example:
https://github.com/google/transit/blob/master/gtfs-realtime/spec/en/reference.md#message-feedmessage

Reply:
But we don't need "Cardinality" here